charts/umbrella/values.yaml

```
@@ -649,3 +649,21 @@ dataprovider:
     nameOverride: dataprovider-submodelserver
     ingress:
       enabled: false
+
+semantic-hub:
+  enabled: true
+  enableKeycloak: false
+  hub:
+    authentication: false
+    livenessProbe:
+      initialDelaySeconds: 200
+    readinessProbe:
+      initialDelaySeconds: 200
+  graphdb:
+    enabled: true
+    image: jena-fuseki-docker:4.7.0
+    imagePullPolicy: Always
```

This should be set to never to use the locally built image.

```suggestion
    imagePullPolicy: Never
```